libs/external-router/src/config.ts

```diff
@@ -0,0 +1,26 @@
+import { InjectionToken } from '@angular/core';
+
+const daffExternalRouterConfigurationDefault: DaffExternalRouterConfiguration = {
+	failedResolutionPath: '/',
+};
+
+/**
+ * The token holding the runtime configuration for the behavior of the
+ * `@daffodil/external-router` package.
+ */
+export const DAFF_EXTERNAL_ROUTER_CONFIG = new InjectionToken<
+	DaffExternalRouterConfiguration
+>('DAFF_EXTERNAL_ROUTER_CONFIG', {
+	providedIn: 'root',
```

Comment:
providedIn is 'root' by default, so this can be removed

Reply:
https://github.com/angular/angular/blob/c461acd12e529a253b17ebfae41c4a81fb757707/packages/core/src/di/injection_token.ts#L64

Reply:
Oh, you're right. I missed that. I'd still like to keep the prop though, to make it explicitly clear.